lib/betagouvbot/mailinglistaction.rb

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module BetaGouvBot
+  class MailingListAction
+    DOMAIN = 'beta.gouv.fr'
+    PREFIX = "/email/domain/#{DOMAIN}"
+    def initialize(api, listname, email)
+      @api = api
+      @listname = listname
+      @email = email
+    end
+
+    def to_s
```

Comment:
http://stackoverflow.com/questions/6014900/is-overriding-to-s-methods-in-ruby-bad